recipes/openfoam/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "openfoam" %}
+{% set version = "v2412" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://develop.openfoam.com/Development/openfoam/-/archive/OpenFOAM-{{ version }}/openfoam-OpenFOAM-{{ version }}.tar.gz
+  sha256: 27d06edb4a6ffc6f88a494edd3b289d5d0a0874866a416b6b0d0a26aba2d9c4a
+
+build:
+  skip: True  # [not linux]
+  number: {{ build }}
+  rpaths:
+    - lib/
+    - lib/sys-mpich/
+    - lib/dummy/
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - {{ stdlib("c") }}
+    - make  # [unix]
+    - m2w64-make  # [win]
+    - cmake
+    - mpich
+    - flex  # [unix]
+    - m2-flex  # [win]
+    - zlib
+  host:
+    - scotch
+    - ptscotch
+    - zoltan
+    - kahip
+    - mpich    
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - cgal
+    - fftw
+    - mpfr
+    - ncurses
+    - parmetis
+    - petsc
+    - hypre
+  run:
```

Comment:
Not sure about metis but the other ones should not be explicitly listed here. They should come in automatically as run_exports.

Reply:
removed run requirements (all but metis) it builds just fine as you suspected, thanks for the hint!